scripts/bin-test/run.sh

```diff
@@ -5,10 +5,17 @@ set -ex # Immediately exit on failure
 npm run build
 npm link
 
-# Link local SDK to all test sources.
+# Link the extensions SDKs for the testing environment.
+(cd scripts/bin-test/extsdks/local && npm link)
+(cd scripts/bin-test/extsdks/translate && npm link)
+(cd scripts/bin-test/extsdks/translate && npm link firebase-functions)
+
+# Link SDKs to all test sources.
 for f in scripts/bin-test/sources/*; do
     if [ -d "$f" ]; then
         (cd "$f" && npm link firebase-functions)
+        (cd "$f" && npm link @firebase-extensions/firebase-firestore-translate-text-sdk)
+        (cd "$f" && npm link @firebase-extensions/local-backfill-sdk)
```

Comment:
instead of adding extensions "functions" as a test case on every single set up, can we instead separate out extensions test cases as unique ones?

Reply:
But it's the same if-else code chunk being tested... I don't want someone to "forget" to run half the tests.

Reply:
I don't think anybody runs this test individually but instead runs it via this test script.

I think it'd be cleaner to separate out extension use case - I see it as a specialized use case and isn't representative of most CF3 users set up.